cmd/contour/serve.go

```diff
@@ -385,8 +389,8 @@ type informer interface {
 
 func startInformer(inf informer, log logrus.FieldLogger) func(stop <-chan struct{}) error {
 	return func(stop <-chan struct{}) error {
-		log.Println("starting")
-		defer log.Println("stopped")
+		log.Println("started informer")
```

Comment:
I don't think this is a big deal to add but is that what the `context` in the logger is meant to describe?

Reply:
Yes, but the context is not very obvious unless you are are reading logs with the relevant source code in front of you :)